README.md

```diff
@@ -96,10 +96,10 @@ Users sometimes want to use our Helm charts as sub-charts for other project or i
 
 ## Tobs vs. Kubernetes
 
-| Tobs     | Kubernetes           |
-|----------|----------------------|
-| 0.7.0    | v1.19 to v1.21       | 
-
+| Tobs                  | Kubernetes           |
+|-----------------------|----------------------|
+| 0.8.0 (unreleased)    | v1.21 to v1.23       | 
+| 0.7.0                 | v1.19 to v1.21       | 
```

Comment:
```suggestion
| 0.8.x (unreleased)    | v1.21 to v1.23       | 
| 0.7.x                 | v1.19 to v1.21       | 
```

Let's note that the whole minor versions are compatible with their respective kubernetes versions, not only `.0` patches.